src/app/contacts/shared/contact.service.ts

```diff
@@ -0,0 +1,38 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { IContact } from './contact.model';
+import { HttpClient } from '@angular/common/http';
+import { ToastrService } from 'ngx-toastr';
+
+@Injectable({
+  providedIn: 'root'
+})
+
+export class ContactService {
+  private SERVER_URL = 'http://localhost:3000';
```

Comment:
To bi bilo fajn prestaviti v nek config file.

Reply:
Seveda, gre za nastavitve, ki jih je smiselno imeti na enem mestu in se navadno spreminjajo glede na okolje (dev/test/prod).